stdlib/buffer.gr

```diff
@@ -45,27 +41,30 @@ let _64BIT_LEN = 8
 
 /* Gets the size of a Bytes via its ptr */
 @disableGC
-let getSize = (ptr) => WasmI32.load(ptr, _SIZE_OFFSET)
+let getSize = ptr => WasmI32.load(ptr, _SIZE_OFFSET)
 
 /* Doubles the size of buffer's underlying byte sequence, if the given size is larger than the size of a buffer's underlying byte sequence */
 let autogrow = (len, buf) => {
   while (buf.len + len > Bytes.length(buf.data)) {
     let mut n = Bytes.length(buf.data)
-    if (n == 0) n = 4 // Make sure bytes of 0 length grow too
+    if (n == 0) {
+      n = 4
+    }
+    // Make sure bytes of 0 length grow too
```

Comment:
A little sad this comment got moved.

Reply:
Hmmm, me too.  It shouldn't have got moved there at all.  I'll take a look.

Reply:
Fixed in lated commit